_api-reference/cluster-settings.md

```diff
@@ -26,30 +45,64 @@ PUT _cluster/settings
+action.auto_create_index | Boolean | Automatically creates an index if the index doesn't already exist. Also applies any index templates that are configured. Default is `true`.
+action.destructive_requires_name | Boolean | When set to `true`, you must specify the index name to delete an index. You cannot delete all indexes or use wildcards. Default is `true`.
+cluster.indices.close.enable | Boolean | Enables closing of open indexes in OpenSearch. Default is `true`.
+indices.recovery.max_bytes_per_sec | String | Limits the total inbound and outbound recovery traffic for each node. This applies to peer recoveries and snapshot recoveries. Default is `40mb`.
+indices.recovery.max_concurrent_file_chunks | Integer | The number of file chunks sent in parallel for each recovery operation. Default is `2`.
+indices.recovery.max_concurrent_operations | Integer | The number of operations sent in parallel for each recovery. Default is `1`.
+logger.org.opensearch.discovery | String | Loggers accept Log4j2’s built-in log levels: `OFF`, `FATAL`, `ERROR`, `WARN`, `INFO`, `DEBUG`, and `TRACE`. Default is `INFO`.
+breaker.model_inference.limit | String | The limit for the trained model circuit breaker. Default is `50%` of JVM heap.
+breaker.model_inference.overhead | Integer | The constant that all trained model estimations are multiplied by to determine a final estimation. Default is `1`.
+search.max_buckets | Integer | The maximum number of aggregation buckets allowed in a single response. Default is `65536`.
+snapshot.max_concurrent_operations | Integer | The maximum number of concurrent snapshot operations. Default is `1000`.
+slm.health.failed_snapshot_warn_threshold | String | The number of failed invocations since the last successful snapshot that will indicate a problem as per the health API profile. Default is five repeated failures: `5L`.
+indices.breaker.total.limit | String | The starting limit for the overall parent breaker. Default is `70%` of JVM heap if `indices.breaker.total.use_real_memory` is set to `false`. Default is `95%` of JVM heap if `indices.breaker.total.use_real_memory` is set to `true`. 
+indices.breaker.fielddata.limit | String | The limit for the fielddata breaker. Default is `40%` of JVM heap.
+indices.breaker.fielddata.overhead | Floating point | The constant that all fielddata estimations are multiplied by to determine a final estimation. Default is `1.03`.
+indices.breaker.request.limit | String | The limit for the request breaker. Default is `60%` of JVM heap.
+indices.breaker.request.overhead | Integer | The constant that all request estimations are multiplied by to determine a final estimation. Default is `1`.
+network.breaker.inflight_requests.limit | String | The limit for the inflight requests breaker. Default is `100%` of JVM heap.
+network.breaker.inflight_requests.overhead | Integer/Time unit | The constant that all in flight requests estimations are multiplied by to determine a final estimation. Default is `2`.
```

Comment:
Does this capture the meaning?
"... that all in-flight request estimations are ..."

Reply:
yeah, that would keep the meaning. I'll go ahead and update it.